R/learn.R

```diff
@@ -80,7 +80,11 @@ dm_learn_from_db <- function(dest, ...) {
 
 schema_if <- function(schema, table, con) {
   table_sql <- DBI::dbQuoteIdentifier(con, table)
-  if_else(is.na(schema), table_sql, paste0(DBI::dbQuoteIdentifier(con, schema), ".", table_sql))
+  if (is.na(schema) || (is_mssql(con) && schema == "dbo")) {
```

Comment:
AFAIK, if we are connected to a certain DB on a MSSQL DBMS, the syntax schema.table works, if we want to access tables on that DB. Only if we wanted to access a table on another DB on that DB-server we would need to use the syntax you mentioned.

Good point with the less ambiguous notation. I will revert that commit then and adapt the tests instead.
Only thing is that this will currently only affect MSSQL-DBs when using `dm_from_src()`. On Postgres I did not explicitly name the default schema, since `src_tbls()` works that way anyway on Postgres. Shall I add the default schema explicitly for PG then too?